### ISSUE TITLE
fail to install

### ISSUE THREAD

**Issue by infoxiao** [myint/language-check#30]
Error message:

error in language-check setup command: package_data must be a dictionary mapping package names to lists of wildcard patterns

**Comment by jayvdb** [comment]
Which version of Python was this?
And pushing my luck, do you remember (or can find in a build log) which version of pip and setuptools were installed?

**Comment by jayvdb** [comment]
I reproduced this on Python 2.7, using recent pip&setuptools:

```
$ python2 setup.py develop
error in language-check setup command: package_data must be a dictionary mapping package names to lists of wildcard patterns
$ python2 -m pip install --user .
Obtaining file:///.../language-check
    Complete output from command python setup.py egg_info:
    error in language-check setup command: package_data must be a dictionary mapping package names to lists of wildcard patterns
$ python2 -m pip --version
pip 9.0.1 from /home/jayvdb/.local/lib/python2.7/site-packages (python 2.7)
$ python2 -c "import setuptools; print(setuptools.__version__)"
34.3.0
```

**Comment by myint** [comment]
Does it have anything to do with whether [`3to2`](https://github.com/myint/language-check#installation) is installed?

**Comment by jayvdb** [comment]
No, I installed that manually, as there is a (fairly certain) separate problem related to that.

**Comment by jayvdb** [comment]
A little strange.

https://github.com/jayvdb/language-check/commit/be204a511 fixes the problem for me, by forcing the `unicode` key to be a `str` on Python 2.

But the relevant check was [added](https://github.com/pypa/setuptools/commit/d98923012) to setuptools in Dec 2005.

The only other explanation is that the `value` being parsed in `get_package_data` was already a `str`.
I'll dig a little more now.

**Comment by jayvdb** [comment]
btw there is a lot about the surface problem at http://stackoverflow.com/questions/23174738/setup-py-packages-and-unicode-literals

Worth noting that I am using the Fedora python setuptools package, which is de-vendored.  In case it is relevant .. my `six` is 1.10.0

But we still dont know why I see the problem and Travis works correctly, which is annoying me a bit.

**Comment by jayvdb** [comment]
Ok, the problem is I have [configparser](https://pypi.python.org/pypi/configparser) installed by package `python2-configparser-3.5.0b2-5.fc26`.

If I bypass that, the problem goes away.

**Comment by sjdemartini** [comment]
I just came across this same error (`error in language-check setup command: package_data must be a dictionary mapping package names to lists of wildcard patterns`). Running:
```
pip uninstall configparser
```
and then
```
pip install language-check
```
did the trick for me. Thanks for pointing that out @jayvdb. 

Is a version-bump of `language-check` and re-publish to pypi needed to resolve this for people going forward (given that https://github.com/myint/language-check/pull/37 was merged), or should this be re-opened as a new issue?

**Comment by myint** [comment]
@sjdemartini I've released [version 1.1](https://pypi.python.org/pypi/language-check/1.1).

**Comment by sjdemartini** [comment]
@myint Thanks for the new release! It appears that even on Version 1.1, installing `language-check` for Python 2.7 fails if `configparser` is installed. Got the following error:
```
  Using cached language-check-1.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-dEmwD2/language-check/setup.py", line 595, in <module>
        sys.exit(main())
      File "/tmp/pip-build-dEmwD2/language-check/setup.py", line 591, in main
        setup(**cfg_to_args(config))
      File "/tmp/pip-build-dEmwD2/language-check/setup.py", line 336, in cfg_to_args
        value = get_cfg_value(config, section, option)
      File "/tmp/pip-build-dEmwD2/language-check/setup.py", line 233, in get_cfg_value
        value = config[section][option]
    AttributeError: RawConfigParser instance has no attribute '__getitem__'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-dEmwD2/language-check/
```
A `pip uninstall configparser` proved necessary to get past that.
